csrc/kernel_ir.h

```diff
@@ -586,6 +586,11 @@ class TORCH_CUDA_CU_API ForLoop final : public Expr {
  private:
   //! Returns if a loop could be unrolled.
   bool isUnrollable() const;
+
+  //! Not storing this as an attribute because this is only a cache for
+  //! simplifiedStop. We are not interested in keeping this across clone/serde,
+  //! etc.
+  mutable Val* simplified_stop_ = nullptr;
```

Comment:
This is a good example for when we want to use a private member for an `Expr` instead of using an attribute.